docs/getting-started/github-apps/github-apps.rst

```diff
@@ -82,7 +82,7 @@ name (optional)
 subclass
     The descriptor language used for the workflow. Supported values include CWL, WDL, NFL (Nextflow), and GXFORMAT2 (Galaxy). This cannot be changed once the workflow is registered.
 primaryDescriptorPath
-    The absolute path to the primary descriptor file in the Git repository.
+    The absolute path to the primary descriptor file in the Git repository. For Nextflow, the primary descriptor is the .config file.
```

Comment:
Isn't it typically `nextflow.config`? If so, we should say that.

This feels a little out of place to me. I feel like like we should mention all languages or none. Maybe have sub-bullets? For Galaxy, the primary descriptor is a .ga file; for WDL the primary descriptor file is a .wdl file, etc.

But if I'm the only who feels this way, I'm OK with.

Reply:
👍